serbian/noveIdeje/noveIdeje.txt

```diff
@@ -0,0 +1 @@
+"Ali vec sam ti rekao"-uzviknuo sam ljutito.
```

Comment:
Please make sure you name all of your files with the extension `.md`.  Otherwise, the files will not display correctly on GitHub.